flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -256,6 +254,9 @@ static void uavoFrSKYSensorHubBridgeTask(void *parameters)
 	GPSPositionData gpsPosData;
 	BaroAltitudeData baroAltitude;
 	FlightStatusData flightStatus;
+
+	uint8_t *serial_buf = frsky_state->serial_buf;
```

Comment:
Yah, it was to avoid having to change a bunch of the below lines to frsky_state->serial_buf ; this does likely waste 4 bytes of stack space but only when the module is running (probably the fact that frsky_state doesn't change escapes analysis).

There's a lot of frsky pack calls that would need to be updated but I'm willing to change them if you'd like.